### ISSUE TITLE
File upload field content does not get saved

### ISSUE THREAD

**Issue by EricMgns** [verbb/formie#2174]
### Describe the bug

I have a multipage form where File upload fields are nested into Groups, on back/next/submit the value for this field is never saved, if I look into the submission on CP then no value is set there either on the fields. The asset is successfully uploaded to the volume however.

### Steps to reproduce

1. Create Form
2. Have File Upload Fields within a Group

### Form settings

- Multi-page form: Yes
- Submission Method:  Page Reload
- Client-side Validation: Yes
- Custom Form Templates: No

### Craft CMS version

5.5.5

### Plugin version

3.0.14

**Comment by engram-design** [comment]
Things seem to be working okay on my end. Can you send a support request via Formie > Settings > Get Support? That'll allow me to replicate your form at least.

**Comment by engram-design** [comment]
Thanks for sending that through. I did actually hit some snags when dealing with your conditions, which uncovered another issue, which I've fixed. You were using a checkbox with a numeric value, which wasn't toggling those group fields for me. I believe that may have been the actual issue.

Fixed for the next release. To get this early, run `composer require verbb/formie:"dev-craft-5 as 3.0.14"`.
